packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-token.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { html, LitElement } from 'lit';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+/**
+ * An element used by `<vaadin-multi-select-combo-box>` to display selected items.
+ *
+ * ### Styling
+ *
+ * The following shadow DOM parts are available for styling:
+ *
+ * Part name        | Description
+ * -----------------|-------------
+ * `label`          | Element containing the label
+ * `remove-button`  | Remove token button
+ *
+ * See [Styling Components](https://vaadin.com/docs/latest/ds/customization/styling-components) documentation.
+ *
+ * @extends HTMLElement
+ * @private
+ */
+class MultiSelectComboBoxToken extends ThemableMixin(LitElement) {
+  static get is() {
+    return 'vaadin-multi-select-combo-box-token';
+  }
+
+  static get properties() {
+    return {
+      label: {
+        type: String
+      },
+
+      item: {
+        type: Object
+      }
+    };
+  }
+
+  render() {
+    return html`
+      <div part="label">${this.label}</div>
+      <div part="remove-button" role="button" @click="${this._onRemoveClick}"></div>
+    `;
```

Comment:
What do you think about using vaadin-button here as well, like you did in the password field? This would probably help with the same UX and a11y everywhere. Probably with an aria label "Remove item {name}" as well. 

Totally of topic: while checking the password field Impl, I was wondering if the following lines should be swapped.. I've got the feeling that now the i18n is applied and afterwards it's overwritten by the default again or am I misinterpreting it?

https://github.com/vaadin/web-components/blob/9e02f531489354f8893d8ece9f3f096c9e1ca56e/packages/password-field/src/vaadin-password-field.js#L172-L173

Reply:
IMO we should eventually have a separate component e.g. `vaadin-chip` (see for example [Material Chips](https://material.io/components/chips)) with a focusable button that could use `vaadin-button` similarly to the reveal button.

> the i18n is applied and afterwards it's overwritten by the default again

Thanks for sharing! I will create an issue for it.